varats/varats/experiments/base/time_workloads.py

```diff
@@ -0,0 +1,116 @@
+"""Implements an experiment times the execution of all project binaries."""
+
+import typing as tp
+from pathlib import Path
+
+from benchbuild import Project
+from benchbuild.command import cleanup
+from benchbuild.extensions import compiler, run
+from benchbuild.utils import actions
+from benchbuild.utils.cmd import time
+from plumbum import local
+
+from varats.experiment.experiment_util import (
+    VersionExperiment,
+    get_default_compile_error_wrapped,
+    create_new_success_result_filepath,
+    ZippedExperimentSteps,
+)
+from varats.experiment.workload_util import (
+    workload_commands,
+    WorkloadCategory,
+    create_workload_specific_filename,
+)
+from varats.project.project_util import ProjectBinaryWrapper
+from varats.project.varats_project import VProject
+from varats.report.gnu_time_report import TimeReportAggregate
+from varats.report.report import ReportSpecification
+
+
+class TimeProjectWorkloads(actions.ProjectStep):  # type: ignore
+    """Times the execution of all project workloads."""
```

Comment:
Do you intend to execute all workloads or all example workloads?

```suggestion
    """Times the execution of all project example workloads."""
```